src/components/grid-aware/Modal/Modal.js

```diff
@@ -13,10 +14,18 @@ const Modal = ({
   contentLabel,
   noBezel,
 }) => {
-  const modalClassName = `${s.content} ${noBezel ? s.noBezel : s.withBezel}`;
-  const buttonClassName = `${s.closeButton} ${
-    noBezel ? s.noBezel : s.withBezel
-  }`;
+  const classStyles = classNames.bind(s);
+  const modalClassName = classStyles({
+    content: true,
```

Comment:
According to the `classnames`' documentation, you can mix both the string argument method and the object argument method in the same call, where passing in a string is equivalent to setting a key of that name to "true" in the object method. The example from their docs is:

```js
classNames('foo', { bar: true }); // => 'foo bar'
```

What do you think about

```js
classStyles('content', { noBezel, withBezel: !noBezel })
```
?

I think I could go either way, but whichever style we pick, I think we should exclusively go with that style and make it a repo-wide convention. This would be the perfect kind of thing to put in a style guide, which I've been meaning to write down for this application.

@davidagustin, @ivanleeswe, and @jessypeck, what do you think about the two different approaches?

Reply:
I think I'm fine with using this pattern:
```javascript
classStyles('content', { noBezel, withBezel: !noBezel })
```
Although I do want to say that the two different uses of implicit assignment (`content` via this library, and `noBezel` via the object property value shorthand) initially threw me off for a second. It took me a little extra time to understand why they had to be handled in different ways, and what the final result would look like. That said, I'm fine now that I've looked over the docs. But I do think it's worth observing that by doing more things implicitly, we're asking people to do a little bit more brain gymnastics to understand what's happening.

But looking at the docs, I don't find this example at all difficult to parse:
```javascript
classNames('foo', { bar: true, duck: false }, 'baz', { quux: true });
```
And if anything, I like the way it helps separate the static, always present class names from the conditional ones. So if we as a group think these implicit assignments won't throw us off, I think I'm leaning in this direction.

But I also have no objection to just explicitly declaring everything, even if it might look a little clunkier.

Reply:
Thanks for the feedback! I am also fine with using this pattern.